plugins/VisitsSummary/Controller.php

```diff
@@ -106,6 +107,17 @@ public function getEvolutionGraph(array $columns = array(), array $defaultColumn
             'avg_time_generation'
         );
 
+        $period = Common::getRequestVar('period', false);
+
+        if (SettingsPiwik::isUniqueVisitorsEnabled($period)) {
+            // add number of unique visitors if available
+            $selectableColumns = array_merge(
+                array($selectableColumns[0]),
+                array('nb_uniq_visitors'),
+                array_slice($selectableColumns, 1)
+            );
```

Comment:
I see you point, but that behaviour was simply "copied" from another place.
Imho we should more likely discuss if we should introduce some kind of column ordering system that will be applied to all reports automatically. We could define that in the column classes or maybe have kind of a registry where columns can register and define there order, which would maybe make it easier for third party plugin to work with it. Maybe that could also be used to define a custom order for any user... 
@tsteur @mattab any plans to do something like that in 3.0?